README.md

```diff
@@ -1 +1,197 @@
+  - [ ] Book: Strategic Monoliths and Microservices by Vaughn Vernon
+  - [ ] Conferences
+  - [ ] Blogs and Online Resources
+  - [ ] Workshops and Trainings
+  - [ ] Others... (open text)
+
+### General DDD Concepts
+
+- Which is the most important thing about DDD? It's too open in purpose, there's no right and wrong answer. (Open Text)
+
+- What's a Domain Model? (Open Text)
+
+- What's Domain Modelling? (Open Text)
+
+- In which scenario you would use DDD? (Open Text)
+
+- In which scenario you would NOT use DDD? (Open Text)
+
+- When do you know what you're doing DDD wrong? (Open Text)
+
+- DDD helps you in (Multiple choices)
+  - [ ] Manage Business Complexity
+  - [ ] Mange Technical Complexity
+  - [ ] Define good Microservices
+  - [ ] Have good Architecture because of Tactical Patterns
+  - [ ] Other... (Open text)
+
+- What's a Big Ball of Mud? (Open text)
+
+### Strategic Design
+
+- Which Strategic Patterns are you familiar with? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Other... (Open text)
+
+- Which of those Patterns are used for the Problem Space? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Microservices
+  - [ ] Other... (Open text)
+
+- Which of those Patterns are used for the Solution Space? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Microservices
+  - [ ] Other... (Open text)
+
+- How do you identify a Bounded Context? (Open text)
+
+- How do you know what's Core Domain? (Open text)
+
+- How do you know what's Supportive Subdomain? (Open text)
+
+- How do you know what's Generic Subdomain? (Open text)
+
+- How do you differentiate between them? (Open text)
+
+- What's Context Mapping used for? (Open text)
+
+- Which kind of Context Mappings do you know? (Open text)
+
+### Domain Modelling with Event Storming
+
+- Are you familiar with Event Storming? (Choose one)
+  - ( ) No 
+  - ( ) I read about it
+  - ( ) I participated in few (< 10)
+  - ( ) I participated in several sessions ( >= 10)
+  - ( ) I facilitated several sessions ( assume here you also participated in several )
+
+- What's needed to run an Event Storming? (Open text)
+
+- What's the main goal of an Event Storming? (Open text)
+
+- How does Event Storming relate with Strategic Design and Tactical Design? (Open text)
+
+### Tactical Design
+
+- Why Hexagonal Architecture is so popular within DDD? (Open text)
+
+- Which patterns did you use? (Multiple choices)
+  - [ ] Domain Service
+  - [ ] Domain Event
+  - [ ] Aggregate
+  - [ ] Repository
+  - [ ] Entity
+  - [ ] Value Object
+  - [ ] CQRS
+  - [ ] Event Sourcing
+  - [ ] Other... (Open text)
+
+- What is a Domain Service? (Open text)
```

Comment:
I think these questions could be softened a little. Also, looking back on the other sections, the questions that seem to be more useful are those with are more of a self-assessment or experience-based.

With that in mind, maybe you could rephrase these questions to be of a format like: 

"When have you used a domain service?"